doc/src/guides/solving/solve-polynomial.md

```diff
@@ -0,0 +1,278 @@
+# Solve (Find the Roots of) a Polynomial Algebraically 
+
+Use SymPy to solve (find the roots of) a polynomial algebraically. For example,
+solving $ax^2 + bx + c = 0$ for $x$ yields $x = \frac{-b\pm\sqrt{b^2 -
+4ac}}{2a}$.
+
+Alternatives to consider:
+- If you need a numeric (rather than algebraic) solution, you can use either
+    - NumPy's {external:func}`~numpy.roots`
+    - SciPy's {external:func}`~scipy.optimize.root`
+
+Here is an example of solving a polynomial algebraically:
+
+```py
+>>> from sympy import solve
+>>> from sympy.abc import x, a, b, c
+>>> solve(a*x**2 + b*x + c, x, dict=True)
+[{x: (-b - sqrt(-4*a*c + b**2))/(2*a)}, {x: (-b + sqrt(-4*a*c + b**2))/(2*a)}]
+```
+
+This example reproduces the [quadratic
+formula](https://en.wikipedia.org/wiki/Quadratic_formula).
+
+## Guidance
+
+### *Guidance 1*
+
+*Guidance 1 content*
+
+### *Guidance 2*
+
+*Guidance 2 content*
+
+
+## Solve (Find the Roots of) a Polynomial
+
+You can solve a polynomial algebraically in several ways. The one to use depends
+on whether you
+- want an algebraic or numeric answer
+- want the multiplicity of each root (how many times each root is a solution).
+  In the `expression` below representing $(x+2)^2(x-3)$, the root -2 has a
+  multiplicity of two because $x+2$ is squared, whereas 3 has a multiplicity of
+  one because $x-3$ has no exponent. Similarly, for the `symbolic` expression,
+  the root $-a$ has a multiplicity of two and the root $b$ has a multiplicity of
+  one.
+
+```py
+>>> from sympy import solve, roots, real_roots, factor, nroots, RootOf, expand
+>>> from sympy import Poly
+>>> from sympy.abc import x, a, b
+>>> expression = (x+2)**2 * (x-3)
+>>> symbolic = (x+a)**2 * (x-b)
+```
+
+### Algebraic Solution Without Root Multiplicities
+
+You can use SymPy's standard {func}`~.solve` function, though it will not return
+the multiplicity of roots:
+
+```py
+>>> solve(expression, x, dict=True)
+[{x: -2}, {x: 3}]
+>>> solve(symbolic, x, dict=True)
+[{x: -a}, {x: b}]
+```
+
+Refer to [](solve-equation-algebraically.md) for more about using
+{func}`~.solve`.
+
+### Algebraic Solution With Root Multiplicities
+
+#### {func}`~.roots`
+
+{func}`~.roots` is the most rigorous function because it can give explicit
+expressions for the roots of polynomials that have symbolic coefficients (that
+is, if there are symbols in the coefficients) if {func}`~.factor` does not
+reveal them. However, it may fail for some polynomials. Here are examples of
+{func}`~.roots`:
```

Comment:
I would not describe `roots` as the most rigorous. Both `factor` and `RootOf` have much better claim to that description.

Also `roots` can sometimes return only a subset of the roots or nothing if it can't express any roots in radicals (although `strict=True` can be used for this):
```python
In [2]: roots(x**5 - x + 1, x)
Out[2]: {}

In [5]: roots((x**5 - x + 1)*x, x)
Out[5]: {0: 1}

In [3]: roots(x**5 - x + 1, x, strict=True)
---------------------------------------------------------------------------
UnsolvableFactorError
```